common/lib/xmodule/xmodule/wrapper_module.py

```diff
@@ -9,14 +9,7 @@
 class_priority = ['video', 'problem']
 
 
-class WrapperModule(VerticalModule):
-    ''' Layout module for laying out submodules vertically w/o css delimiters'''
-
-    has_children = True
-    css = {'scss': [resource_string(__name__, 'css/wrapper/display.scss')]}
-
-
-class WrapperDescriptor(VerticalDescriptor):
-    module_class = WrapperModule
+class WrapperBlock(VerticalBlock):
+    ''' Layout block for laying out sub-blocks vertically w/o css delimiters'''
 
     has_children = True
```

Comment:
How does this class do anything? It's no different than VerticalBlock.  And do you even need the `has_children` line? It will inherit that from VerticalBlock, no?